Packs/CortexXDR/ReleaseNotes/6_1_3.md

```diff
@@ -0,0 +1,11 @@
+
+#### Integrations
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+
+- Improved the following commands results: 
+  - ***xdr-kill-process-script-execute***
+  - ***xdr-file-exist-script-execute***
```

Comment:
and since this one is not a polling command, we can remove it (the effect here is only about the number of entries and the human readable, not really interesting)
```suggestion
```